py_gtfs_rt_ingestion/batch_files.py

```diff
@@ -29,6 +30,7 @@ class BatchArgs(NamedTuple):
     s3_prefix: str = DEFAULT_S3_PREFIX
     print_events: bool = False
     dry_run: bool = False
+    debug_rds_connection: bool = False
```

Comment:
new bool to add to try debugging connections.

not a _huge_ fan of debugging this way, but it was all i could think to do.